src/plugins/intel_gpu/src/kernel_selector/kernels/ctc_greedy_decoder/ctc_greedy_decoder_kernel_ref.cpp

```diff
@@ -37,4 +37,14 @@ KernelsData CTCGreedyDecoderKernelRef::GetKernelsData(const Params& params, cons
 KernelsPriority CTCGreedyDecoderKernelRef::GetKernelsPriority(const Params& /*params*/, const optional_params& /*options*/) const {
     return FORCE_PRIORITY_9;
 }
+
+JitConstants CTCGreedyDecoderKernelRef::GetJitConstants(const ctc_greedy_decoder_params& params, DispatchData dispatchData) const {
+    auto jit = CTCGreedyDecoderKernelBase::GetJitConstants(params, dispatchData);
+
+    if (params.use_multiple_outputs) {
+        jit.AddConstant(MakeJitConstant("MULTIPLE_OUTPUTS", 1));
```

Comment:
I wonder that existing cl kernel code already have option of MULTIPLE_OUTPUTS.
If yes, how does it work for static shape case?

Reply:
Static shape case uses SECOND_OUTPUT not MULTIPLE_OUTPUTS.

Reply:
I was wondering that you added new jit constant without cl kernel change. It was added in second commit.

Reply:
Yes, cl kernel works same for static and dynamic shapes. So dynamic shapes case added "MULTIPLE_OUTPUTS" only.

Reply:
This will make things confusing in the future, how about changing the jitconfigs like 
- SECOND_OUTPUT => LEGACY_MULTIPLE_OUTPUTS
- MULTIPLE_OUTPUTS => NEW_MULTIPLE_OUTPUTS 

? (naming to be aligned with allow_new_shape_infer)

Reply:
SECOND_OUTPUT_EXIST and MULTIPLE_OUTPUTS are used in other kernels as well: arg_max_min_axis.cl and non_max_suppression_gpu_ref.cl.
And SECOND_OUTPUT_EXIST has other forms of macros: SECOND_OUTPUT_GET_INDEX and SECOND_OUTPUT_TYPE.
Did you mean to change them all?

Reply:
Of course we don't need to fix all of them at this moment, but I'd like to have better fix for new commits, as possible as we can do . If this is not very urgent commit, could you consider about this?  :)

Reply:
Fixed SECOND_OUTPUT_EXIST => LEGACY_MULTIPLE_OUTPUTS and MULTIPLE_OUTPUTS => NEW_MULTIPLE_OUTPUTS as your comments. Thanks for your comments.